### PR TITLE
Colors

### PR DESCRIPTION
**Pull request by glebec**
I like colors.

I think this could be coded slightly more robustly/elegantly — importing a config file of color choices, instead of hard-coding them, plus not repeating the `chalk.yellow` pattern in each level — but I figured I'd at least see how you feel about adding color to GitFun rather than sink too much time into it.

### CODE REVIEW

**Comment by Kmacpher** [comment]
Ooh, yes I like the colors. Thank you for implementing this.

File: libs/game.js [hunk 1]
```diff
@@ -75,7 +76,7 @@ function check(levelObj) {
 }
 
 function correct(levelObj) {
-  console.log("\nWooo! You completed this git challenge! Run `Gitfun` to proceed to the next level.\n");
+  console.log(chalk.green("\nWooo! You completed this git challenge! ") + "Run `Gitfun` to proceed to the next level.\n");
```
Comment (glebec) [inline libs/game.js:79]: I just noticed another typo here — not urgent but you might want to hotfix it. `Gitfun` -> `gitfun`

Comment (Kmacpher) [reply libs/game.js:79]: You actually fixed that one in your other PR :)

Comment (glebec) [reply libs/game.js:79]: LOL. Left hand / right hand etc. etc.

**Pull request merged**